project-management-application-app/src/test/java/com/newklearz/controllers/TicketRankControllerIT.java

```diff
@@ -100,4 +156,74 @@ public void testUpdateTicketPriority()
         assertEquals(ticketDTOS.size(), ticketsForBoard.getBody().size());
         assertEquals(ticketsForBoard.getBody().get(1).getId(),ticketRankDTOS.get(2).getAssignedTicket().getId());
     }
+
+    @Test
+    public void testUpdateTicketPriorityNegative()
+    {
+        TicketRankDTO testTicketRank = new TicketRankDTO();
+        testTicketRank.setAssignedTicket(ticketDTOS.get(0));
+        testTicketRank.setAssignedBoard(boardDTO);
+
+        ResponseEntity<TicketRankDTO> ticketRank = ticketRankController.addTicketToBoard(boardDTO.getId(),testTicketRank);
+        ticketRankDTOS.add(ticketRank.getBody());
+        assertNotNull(ticketRank);
+        assertEquals(ticketRank.getStatusCode(), HttpStatus.OK);
+
+        TicketRankDTO testTicketRank3 = new TicketRankDTO();
+        testTicketRank3.setAssignedTicket(ticketDTOS.get(2));
+        testTicketRank3.setAssignedBoard(boardDTO);
+
+        ResponseEntity<TicketRankDTO> ticketRank3 = ticketRankController.addTicketToBoard(boardDTO.getId(),testTicketRank3);
+        ticketRankDTOS.add(ticketRank3.getBody());
+        assertNotNull(ticketRank3);
+        assertEquals(ticketRank3.getStatusCode(), HttpStatus.OK);
+
+        /**
+         * Update rank priority with outOfBounds positions
+         */
+        ResponseEntity<Object> outOfBoundsPositions = ticketRankController.updateTicketPriority(boardDTO.getId(),1231231231,123123123);
+        assertEquals(HttpStatus.BAD_REQUEST, outOfBoundsPositions.getStatusCode());
+
+        /**
+         * Update rank priority with negative positions
+         */
+        ResponseEntity<Object> negativeArrayPositions = ticketRankController.updateTicketPriority(boardDTO.getId(),Integer.MAX_VALUE + 3488799,Integer.MAX_VALUE + 3488797);
+        assertEquals(HttpStatus.BAD_REQUEST, negativeArrayPositions.getStatusCode());
```

Comment:
same as above, use Integer.MAX/MIN or LONG.MAX/MIN or Double.MAX/MIN depending on the case